lua/telescope/_extensions/lazy/picker.lua

```diff
@@ -84,9 +84,13 @@ function M.lazy_plugins_picker()
 
   local function attach_mappings(prompt_bufnr, map)
     lazy_actions.default_action_replace(prompt_bufnr)
-    map("i", opts.mappings.open_in_browser, lazy_actions.open_in_browser)
-    map("i", opts.mappings.open_in_find_files, lazy_actions.open_in_find_files)
-    map("i", opts.mappings.open_in_live_grep, lazy_actions.open_in_live_grep)
+
+    for _, mode in ipairs({ "i", "n" }) do
+      map(mode, opts.mappings.open_in_browser, lazy_actions.open_in_browser)
```

Comment:
No need for a for loop here.
Just directly add `map({ "i", "n" }, ...`.

Make sure to also add the same in `actions.lua` there is the mapping to re-open the plugins picker.

Reply:
There actually is a need for a loop here. The `map` callback that's getting passed doesn't support passing a table, they even multiply code in the telescope's source, here is a sample from "Git Branches" picker:
```lua
        map("i", "<c-t>", actions.git_track_branch)
        map("n", "<c-t>", actions.git_track_branch)

        map("i", "<c-r>", actions.git_rebase_branch)
        map("n", "<c-r>", actions.git_rebase_branch)

        map("i", "<c-a>", actions.git_create_branch)
        map("n", "<c-a>", actions.git_create_branch)

        map("i", "<c-s>", actions.git_switch_branch)
        map("n", "<c-s>", actions.git_switch_branch)

        map("i", "<c-d>", actions.git_delete_branch)
        map("n", "<c-d>", actions.git_delete_branch)

        map("i", "<c-y>", actions.git_merge_branch)
        map("n", "<c-y>", actions.git_merge_branch)
```
And sure, I'll put that in `actions.lua` right away.

Reply:
I'm not sure what sample you're looking at but `map` function parameter already accepts a table with modes as the first argument. You can check that also in the documentation of telescope 😄

Reply:
https://github.com/nvim-telescope/telescope.nvim/blob/master/doc/telescope.txt#L1797

Reply:
Alright, done